packages/x-charts/src/ScatterChart/Scatter.tsx

```diff
@@ -115,15 +122,16 @@ function Scatter(props: ScatterProps) {
   return (
     <g>
       {cleanData.map((dataPoint) => (
-        <circle
+        <ScatterMarkerElement
```

Comment:
We previously used `circle` here for performance reasons, as it is much faster, and is specially faster when there are a lot of data. We might want to find a compromise between custom and default, where the default is fast, the custom is up to the user 😄 

Eg:

```tsx
const props = {...}

return React.createElement(slots.marker ?? 'circle', props)
```

Reply:
Make sense 👍 I'll focus on performance after I address the rest of the comments